engine/sound/src/test/test_sound.cpp

```diff
@@ -1439,7 +1481,7 @@ const TestParams2 params_mixer_test[] = {
                 2048,
                 true),
 };
-INSTANTIATE_TEST_CASE_P(dmSoundMixerTest, dmSoundMixerTest, jc_test_values_in(params_mixer_test));
+//INSTANTIATE_TEST_CASE_P(dmSoundMixerTest, dmSoundMixerTest, jc_test_values_in(params_mixer_test));
```

Comment:
Why commented out?

Reply:
oops, my bad! 

I commented all of the test cases except for the one I worked on. The one above slipped form re-enablement. :-(.

Let me enable it...

Btw, @JCash , is there a way to specify only a specific test case to run ? I  recall there was a small bug in the test library preventing this.

Reply:
I've since fixed that issue in jc_test.h

Use the `--test-filter Substring` to run those tests